### PR TITLE
Bring back option to set offer message.

### PR DESCRIPTION
**Pull request by idinium96**

### CODE REVIEW

**Comment by Nicklason** [comment]
Please only work in the master branch so that we don't get a lot of different changed files that has nothing to do with what you are adding

File: src/classes/Cart.ts [hunk 1]
```diff
@@ -215,7 +215,7 @@ abstract class Cart {
 
         this.offer.data('handleTimestamp', moment().valueOf());
 
-        this.offer.setMessage('Powered by TF2 Automatic');
+        this.offer.setMessage('Powered by TF2 Automatic. ' + process.env.OFFER_MESSAGE || 'Powered by TF2 Automatic');
```
Comment (Nicklason) [inline src/classes/Cart.ts]: If there is no message, it will say `Powered by TF2 Automatic. Powered by TF2 Automatic`

Comment (idinium96) [reply src/classes/Cart.ts]: I've tried put the brackets before, but this CLI checking things rejected it. That one I've tried and done have that problem.
checkerror

**Comment by idinium96** [comment]
Well @Nicklason .
pull request

File: src/classes/Cart.ts (hunk 1 above)
Comment (Nicklason) [reply src/classes/Cart.ts]: You are not using prettier, just use that and it will be fixed

**Comment by Nicklason** [comment]
> Well, @Nicklason. You told me to do this way. I do it. #331
> pull request

Yes, but you should only make pull requests to and from the development branch.

**Comment by Nicklason** [comment]
Closing so you can try again using the proper tools and branches required.

**Pull request closed without merging**